.docker/Dockerfile

```diff
@@ -16,10 +16,10 @@ RUN VERSION=$(grep -A 1 'BUNDLED WITH' Gemfile.lock | tail -n 1 | xargs); \
 RUN apt update && apt install -y python3 python3-pip
 
 # install python packages
-RUN python3 -m pip install --upgrade --requirement requirements.txt
+RUN python3 -m pip install --no-cache-dir --upgrade --requirement requirements.txt
 
 # install python package for listening for file changes
-RUN pip install "watchdog[watchmedo]"
+RUN pip install "watchdog[watchmedo]==3.0.0"
```

Comment:
One of my goals in this project is to keep the number of files as minimal as possible. Less for the user to get confused by, tamper with, etc. This also seems like it would necessitate some changed or extra commands to make use of this toml file. To me this is too much added complexity for such little benefit. If we were installing several packages in just the docker I'd reconsider.

Reply:
Thanks, completely understand, and I trust your judgement here.